### ISSUE TITLE
pre-push tests fail when downloading mongodb

### ISSUE THREAD

**Issue by mrtndwrd** [RocketChat/Rocket.Chat#17368]
### Description:

Before I can push any change to a repo, the `pre-push` hook runs `meteor npm run testunit`. This fails on my machine for several reasons.

### Steps to reproduce:

1. Clone the Rocket.Chat repository
2. Run `meteor npm run testunit`
3. ` 1) Sessions Aggregates` tests fail (logs at the bottom of this issue)

### Expected behavior:

Tests succeed (at least on the master branch)

### Actual behavior:

Tests fail, see logs:

[tests-normal.txt](https://github.com/RocketChat/Rocket.Chat/files/4504333/tests-normal.txt)

I tried `apt-get`ing mongodb-server-core (v3.6.3), setting the `MONGO_URL` to my own server (mongodb://localhost:27017), then the tests fail with this output:

[tests-local-mongo.txt](https://github.com/RocketChat/Rocket.Chat/files/4504331/tests-local-mongo.txt)

### Server Setup Information:

- Version of Rocket.Chat Server: current master (0a16a86bee200e390a1361b3d6d8ac6ad44eaf1d / 3.1.1)
- Operating System: Ubuntu 18.04
- Deployment Method: n/a
- Number of Running Instances: n/a
- DB Replicaset Oplog: n/a
- NodeJS Version: v13.11.0
- MongoDB Version: v3.6.3

**Comment by mrtndwrd** [comment]
I worked around the issue by changing the mongoDB version in `./app/models/server/models/Sessions.tests.js` to `'4.0.5'`. 

I got the idea from this issue: https://github.com/mongodb-js/mongodb-download/issues/36

It's possible that I'm the only one running into this issue because I'm on Ubuntu 18.04 and other devs are not?

It seems there are two possible solutions:

1. use a newer version of Mongodb for these tests
2. Wait for mongodb-download to fix the issue and upgrade the dependency
